.github/workflows/release.yml

```diff
@@ -116,7 +116,9 @@ jobs:
         AWS_DEFAULT_REGION: ${{ vars.AWS_DEFAULT_REGION }}
         DSYM_S3_PATH: s3://${{ vars.DSYM_BUCKET_NAME }}/${{ vars.DSYM_BUCKET_PREFIX }}/
       run: |
-        aws s3 cp "${{ github.workspace }}/DuckDuckGo-${{ env.app_version }}-dSYM.zip" ${{ env.DSYM_S3_PATH }}
+        if [[ -f ${{ env.dsyms_path }} ]]; then
```

Comment:
This follows a similar pattern as uploading to Asana on the line 102 in this file.